frontend/public/components/image-stream.tsx

```diff
@@ -325,85 +340,130 @@ export const ImageStreamsDetailsPage: React.FCC = (props) => (
 );
 ImageStreamsDetailsPage.displayName = 'ImageStreamsDetailsPage';
 
-const tableColumnClasses = [
-  '',
-  '',
-  'pf-m-hidden pf-m-visible-on-md',
-  'pf-m-hidden pf-m-visible-on-lg',
-  Kebab.columnClass,
+const tableColumnInfo = [
+  { id: 'name' },
+  { id: 'namespace' },
+  { id: 'labels' },
+  { id: 'created' },
+  { id: 'actions' },
 ];
 
-const ImageStreamsTableRow: React.FC<RowFunctionArgs<K8sResourceKind>> = ({ obj }) => {
-  return (
-    <>
-      <TableData className={tableColumnClasses[0]}>
-        <ResourceLink
-          kind={ImageStreamsReference}
-          name={obj.metadata.name}
-          namespace={obj.metadata.namespace}
-        />
-      </TableData>
-      <TableData className={css(tableColumnClasses[1], 'co-break-word')} columnID="namespace">
-        <ResourceLink kind="Namespace" name={obj.metadata.namespace} />
-      </TableData>
-      <TableData className={tableColumnClasses[2]}>
-        <LabelList kind={ImageStreamsReference} labels={obj.metadata.labels} />
-      </TableData>
-      <TableData className={tableColumnClasses[3]}>
-        <Timestamp timestamp={obj.metadata.creationTimestamp} />
-      </TableData>
-      <TableData className={tableColumnClasses[4]}>
-        <ResourceKebab actions={menuActions} kind={ImageStreamsReference} resource={obj} />
-      </TableData>
-    </>
-  );
+const getDataViewRows: GetDataViewRows<K8sResourceKind, undefined> = (data, columns) => {
+  return data.map(({ obj: imageStream }) => {
+    const { name, namespace, labels, creationTimestamp } = imageStream.metadata;
+
+    const rowCells = {
+      [tableColumnInfo[0].id]: {
+        cell: <ResourceLink kind={ImageStreamsReference} name={name} namespace={namespace} />,
+        props: { ...getNameCellProps(name), width: 10, modifier: 'nowrap' },
```

Comment:
Modifier should be `breakWord`

<img width="1065" height="557" alt="Image" src="https://github.com/user-attachments/assets/a4d9842b-283f-4dd9-91b4-0446c0aa8922" />

Reply:
Width `15` or `20` would give those column names better readability.

Width `10`
<img width="1280" height="1025" alt="Image" src="https://github.com/user-attachments/assets/8aef56bb-f862-48df-979a-1e3cc4e421b3" />